### PR TITLE
Feature/charts on the coins page are not working

### PR DESCRIPTION
**Pull request by AndrewMatsiash**

### CODE REVIEW

File: assets/js/helpers.js [hunk 1]
```diff
@@ -99,3 +99,12 @@ function getCookie(name) {
...
 }
+
+function formatDate(dateString) {
+	const date = new Date(dateString);
+	const day = String(date.getUTCDate()).padStart(2, '0');
+	const month = String(date.getUTCMonth() + 1).padStart(2, '0');
+	const year = String(date.getUTCFullYear()).slice(-2);
+	return `${day}.${month}.${year}`;
+}
+
```
Comment (nnslvp) [inline assets/js/helpers.js:110]: EOF

File: assets/js/api.js [hunk 2]
```diff
@@ -16,66 +21,78 @@ function statsApiPost(action) {
 	})
 }
 
-function fetchCurrencyInfo(coin = 'alephium') {
+function fetchCurrencyInfo(coin = DEFAULT_COIN) {
```
Comment (nnslvp) [inline assets/js/api.js]: DEFAULT_COIN -  не нужен

File: assets/js/helpers.js (hunk 1 above)
Comment (AndrewMatsiash) [reply assets/js/helpers.js:110]: сделал

File: assets/js/api.js (hunk 2 above)
Comment (AndrewMatsiash) [reply assets/js/api.js]: убрал

**Pull request merged**